docs/faq.md

```diff
@@ -0,0 +1,13 @@
+## Frequently Asked Questions
+
+
+Here are the answers to several commonly asked questions:
+
+1. <details><summary>When adding a new binary, the `make builder TARGETS=readme` command fails, claiming it can't find the binary file, even though it is in the expected place.</summary> 
```

Comment:
This error should not be triggered by making `readme`. It should only happen on `make -C vendor/<app> apk`.

Reply:
fixed by #925